packages/api-reference/src/hooks/useDarkModeState.ts

```diff
@@ -46,13 +49,17 @@ export function useDarkModeState(isDarkInitially?: boolean) {
     }
   }
 
-  // Priority of initial values is: LocalStorage/App Config/Fallback
-  isDark.value =
-    (typeof window === 'undefined'
-      ? null
-      : JSON.parse(window.localStorage?.getItem('isDark') || 'null')) ??
-    isDarkInitially ??
-    getDarkModeState()
+  // Priority of initial values is: forceState -> LocalStorage/App ->  Config/Fallback
+  if (forceDarkModeState) {
+    isDark.value = forceDarkModeState === 'dark' ? true : false
+  } else {
+    isDark.value =
+      (typeof window === 'undefined'
+        ? null
+        : JSON.parse(window.localStorage?.getItem('isDark') || 'null')) ??
+      isDarkInitially ??
+      getDarkModeState()
+  }
```

Comment:
this bit looks a bit messy and is hard to follow, maybe re-write?

Reply:
this existed before the PR